Update CLI docs for mainnet acp 77 (#1990)
ava-labs/avalanche-docs: content/docs/avalanche-l1s/deploy-a-avalanche-l1/avalanche-mainnet.mdx

This tutorial shows how to do the following on `Mainnet`.

> - Deploy an Avalanche L1.

Reviewer: Isn't the default still using Subnet-EVM?

Deploy the Avalanche L1[​](#deploy-the-avalanche-l1 "Direct link to heading")
-----------------------------------------------------------------

<Callout title="Note">

> To deploy the Avalanche L1, you will need some AVAX on the P-chain.

Reviewer: ```suggestion
To deploy the Avalanche L1, you will need some AVAX on the P-Chain.
```

```bash
Use the arrow keys to navigate: ↓ ↑ → ←
? Choose a network to deploy on:
  ▸ Local Network
    Fuji

>     Mainnet

Reviewer: Should the arrow point to `Mainnet`?

Ledger address: P-avax1ucykh6ls8thqpuwhg3vp8vvu6spg5e8tp8a25j
```

> Select `No` to using local machine as a bootstrap validator on the blockchain.

Reviewer: ```suggestion
Select `No` when asked if you want to use local machine as a bootstrap validator on the blockchain.
```

Well done. You have just created your own Avalanche L1 on `Mainnet`.

You will be able to see information on the deployed L1 at the end of `avalanche blockchain deploy` command:

```bash
+--------------------+----------------------------------------------------+
| DEPLOYMENT RESULTS |                                                    |
+--------------------+----------------------------------------------------+
| Chain Name         | testblockchain                                     |
+--------------------+----------------------------------------------------+
| Subnet ID          | 2cNuyBhvAd4jH5bFSGndezhB66Z4UHYAsLCMGoCpvhXVhrZfgd |
+--------------------+----------------------------------------------------+
| VM ID              | qcvkEX1zWSz7PtGd7CKvPRBqLVTzA7qyMPvkh5NMDWkuhrcCu  |
+--------------------+----------------------------------------------------+
| Blockchain ID      | 2U7vNdB78xTiN6QtZ9aetfKoGtQhfeEPQG6QZC8bpq8QMf4cDx |
+--------------------+                                                    +
| P-Chain TXID       |                                                    |
+--------------------+----------------------------------------------------+
```

> To get your new Avalanche L1 information, visit the [Avalanche L1 Explorer](https://subnets-test.avax.network/). The search best works by blockchain ID, so in this example, enter `2U7vNdB78xTiN6QtZ9aetfKoGtQhfeEPQG6QZC8bpq8QMf4cDx` into the search box and you should see your shiny new blockchain information.

Reviewer: We should also include `avalanche blockchain describe` as an option for printing blockchain information

To add a validator to an Avalanche L1, the owner of the key that acts as the controller of ValidatorManager contract specified in `avalanche blockchain create` command above run:

```bash
avalanche blockchain addValidator testblockchain
```

Choose `Mainnet`:

```bash
Use the arrow keys to navigate: ↓ ↑ → ←
? Choose a network to deploy on:
  ▸ Mainnet
```

The CLI will show the Ledger address that will be used to pay for add validator tx:

```bash
Ledger address: P-avax1ucykh6ls8thqpuwhg3vp8vvu6spg5e8tp8a25j
```

Now enter the **NodeID** of the new validator to be added.

```bash
What is the NodeID of the validator you'd like to whitelist?: NodeID-BFa1paAAAAAAAAAAAAAAAAAAAAQGjPhUy
```

Next, enter the node's BLS public key and proof of possession.

Now, enter the amount of AVAX that you would like to allocate to the new validator.

The validator's balance is used to pay for continuous fee to the P-Chain. When this Balance reaches 0, the validator will be considered inactive and will no longer participate in validating the L1.

1 AVAX should last the validator about a month.

```bash
What balance would you like to assign to the validator (in AVAX)?: 1
```

> Sign the add validator TX on Ledger:

Reviewer: ```suggestion
Sign the addValidatorTx with your Ledger:
```
